### DISCUSSION TITLE
OpenIDDict: How to Configure token lifetimes through orchard

### DISCUSSION THREAD

**Discussion by s4s-npound** [OrchardCMS/OrchardCore#10845] [Q&A]
So Right now I have a simple orchard setup of services.AddOrchardCms(); and  app.UseOrchardCore();. What can I do to change the token lifetimes for the underlying openiddict tokens? Is that a configuration setting I set or something in startup?  Please let me know.

**Comment by hyzx86** [comment]
FYI.

https://github.com/OrchardCMS/OrchardCore/discussions/7036
